project/Settings.scala

```diff
@@ -38,11 +38,11 @@ object Settings {
     val scalaJQuery  = "1.0"
 
     // Java libraries
-    val scalaZ       = "7.2.2"
+    val scalaZ       = "7.2.10"
     val scalaZStream = "0.8.6a"
```

Comment:
No version bump for `scalaZStream` yet? Do you know if the upgrade is expected for next `http4s` release?